test/README.md

```diff
@@ -45,7 +45,7 @@ To run [the e2e tests](./e2e) and [the conformance tests](./conformance), you ne
 
 ```bash
 go test -v -tags=e2e -count=1 ./test/conformance
-go test -v -tags=e2e -count=1 ./test/e2e
+go test -v -tags=e2e -count=1 -parallel=4 ./test/e2e
```

Comment:
Running tests in parallel should be optional. I suggest you mention it in the docs, but avoid repeating it every time.